actions/workflows/st2_pkg_e2e_test.yaml

```diff
@@ -28,6 +26,10 @@ input:
 vars:
   - vm_id:
   - vm_windows_id:
+  - vm_windows_info:
```

Comment:
Perhaps might be worth a comment to say to remove these when windows hosts is put back in, as it will make it easier to re-introduce when we've found the root cause.